contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/__init__.py

```diff
@@ -15,41 +15,45 @@
 import threading
 
 from opencensus.ext.azure.metrics_exporter import MetricsExporter
-from opencensus.ext.azure.metrics_exporter.heartbeat_metrics.heartbeat import (
-    HeartbeatMetric,
+from opencensus.ext.azure.metrics_exporter.statsbeat_metrics.statsbeat import (
+    _STATS_CONNECTION_STRING,
+    _StatsbeatMetrics,
 )
 from opencensus.metrics import transport
 from opencensus.metrics.export.metric_producer import MetricProducer
 
-_HEARTBEAT_METRICS = None
-_HEARTBEAT_LOCK = threading.Lock()
+_STATSBEAT_METRICS = None
+_STATSBEAT_LOCK = threading.Lock()
 
 
-def enable_heartbeat_metrics(connection_string, ikey):
-    with _HEARTBEAT_LOCK:
-        # Only start heartbeat if did not exist before
-        global _HEARTBEAT_METRICS  # pylint: disable=global-statement
-        if _HEARTBEAT_METRICS is None:
+def collect_statsbeat_metrics(ikey):
+    with _STATSBEAT_LOCK:
+        # Only start statsbeat if did not exist before
+        global _STATSBEAT_METRICS  # pylint: disable=global-statement
+        if _STATSBEAT_METRICS is None:
             exporter = MetricsExporter(
-                connection_string=connection_string,
-                instrumentation_key=ikey,
-                export_interval=900.0,  # Send every 15 minutes
+                is_stats=True,
+                connection_string=_STATS_CONNECTION_STRING,
+                export_interval=5.0,  # Send every 15 minutes
```

Comment:
why is it 5? i guess it's for testing? 
i strongly recommend you make internal configurable via config or env var.